code/game/gamemodes/miniantags/morph/spells/reproduce.dm

```diff
@@ -0,0 +1,42 @@
+/obj/effect/proc_holder/spell/morph/reproduce
+	name = "Reproduce"
+	desc = "Split yourself in half making a new morph. Can only be used while on a floor. Makes you temporarily unable to vent crawl."
+	hunger_cost = 150 // 5 humans
+	charge_max = 30 SECONDS
+	action_icon_state = "morph_reproduce"
+
+
+/obj/effect/proc_holder/spell/morph/reproduce/choose_targets(mob/user)
+	perform(list(user), TRUE, user, FALSE)
+
+/obj/effect/proc_holder/spell/morph/reproduce/cast_check(charge_check = TRUE, start_recharge = TRUE, mob/living/simple_animal/hostile/morph/user = usr)
+	if(!isturf(user.loc))
+		to_chat(user, "<span class='warning'>You can only split while on flooring!</span>")
+		return FALSE
+	return ..()
+
+/obj/effect/proc_holder/spell/morph/reproduce/can_cast(mob/living/simple_animal/hostile/morph/user, charge_check, show_message)
+	. = ..()
+	if(!.)
+		return
+	if(!isturf(user.loc))
+		return FALSE
+
+/obj/effect/proc_holder/spell/morph/reproduce/cast(list/targets, mob/living/simple_animal/hostile/morph/user)
+	to_chat(user, "<span class='sinister'>You prepare to split in two. Making you unable to vent crawl!</span>")
+	user.ventcrawler = FALSE // Temporarily disable it
+	var/list/candidates = SSghost_spawns.poll_candidates("Do you want to play as a morph?", ROLE_MORPH, TRUE, poll_time = 10 SECONDS, source = /mob/living/simple_animal/hostile/morph)
+	if(!length(candidates))
+		to_chat(user, "<span class='warning'>Your body refuses to split at the moment. Try again later.</span>")
+		revert_cast(user)
+		user.ventcrawler = initial(user.ventcrawler) // re enable the crawling
+		return
+	var/mob/C = pick(candidates)
+
+	var/mob/living/simple_animal/hostile/morph/new_morph = new /mob/living/simple_animal/hostile/morph(get_turf(user))
+	var/datum/mind/player_mind = new /datum/mind(C.key)
```

Comment:
I don't think I've ever seen a new mind created like this when used for ghost roles. The observer should already have a mind (hopefully). Is there a reason you did this?

Reply:
If you transfer the mind from the observer to the morph then they will keep their old objectives and memories. Even their martial arts and the like...
I don't want a hijack morph with carp :P

Reply:
That's true I suppose.. which means it already happens with human based ghost roles, abductors, SITs and so on. :flushed: